crates/bevy_app/src/lib.rs

```diff
@@ -42,6 +40,14 @@ pub enum CoreStage {
     /// Name of app stage that runs after all other app stages
     Last,
 }
+
+/// The names of the default App schedules
+#[derive(Debug, Hash, PartialEq, Eq, Clone, StageLabel)]
+pub enum CoreSchedule {
+    /// Runs once at the beginning of the app.
+    Startup,
```

Comment:
Maybe make this a unit type?

Reply:
Didn't know this was possible. I will do this.

One argument against doing this is that the result of #2432 will probably add more schedules, but in that case, the unit type can be converted back to an enum again.

Reply:
IMO, leaving it as an enum is a little nicer since then there's the parallel between `CoreStage` and this. 
And as @MinerSebas said, there's a change this will be expanded upon later.